Classes/Issues/Request/IssueRequestModel.swift

```diff
@@ -42,6 +42,63 @@ final class IssueRequestModel: ListDiffable {
         self.date = date
         self.event = event
 
+        let styledString: StyledTextString
+        if actor == user {
+            styledString = IssueRequestModel.buildSelfString(actor: actor,
+                                                             user: user,
+                                                             date: date,
+                                                             event: event)
+        } else {
+            styledString = IssueRequestModel.buildString(actor: actor,
+                                                         user: user,
+                                                         date: date,
+                                                         event: event)
+        }
+
+        self.string = StyledTextRenderer(
```

Comment:
nit: You don't need `self.` here right?

Reply:
Don't need, but I think is better for consistency since all other members being initialized require `self.` So doing all members even this with it :))

Reply:
They need `self.` because the parameter names match the variable names

Reply:
Yeah, but what I mean by consistency is
```swift
        self.id = id
        self.actor = actor
        self.user = user
        self.date = date
        self.event = event
        string = ... // Also a member 
        // Even it not needing, to maintain consistency in all memeber initialization 
       self.string = ... 

```
But no problem changing if it is a problem :)